pkg/pool-weighted/test/ManagedPoolCircuitBreakerLib.test.ts

```diff
@@ -0,0 +1,154 @@
+import { expect } from 'chai';
+import { Contract, BigNumber } from 'ethers';
+import { deploy } from '@balancer-labs/v2-helpers/src/contract';
+import { fp, fpMul, FP_ZERO, fromFp } from '@balancer-labs/v2-helpers/src/numbers';
+import { random } from 'lodash';
+import { ZERO_BYTES32 } from '@balancer-labs/v2-helpers/src/constants';
+import { sharedBeforeEach } from '@balancer-labs/v2-common/sharedBeforeEach';
+import { CircuitBreakerParams } from '@balancer-labs/v2-helpers/src/models/pools/weighted/types';
+
+describe('ManagedPoolCircuitBreakerLib', () => {
```

Comment:
It looks like we could do a lot of fuzz testing on this library, since we're effectively testing using arbitrary values here.

Reply:
Working on it